src/App.tsx

```diff
@@ -11,11 +11,19 @@ export class App extends React.Component<{}, State> {
   };
 
   addOne = () => {
-    // write code here
+    this.setState((state) => {
+      return {
+        count: state.count + 1,
+      };
+    });
```

Comment:
I would go for a shorthand, same below
```suggestion
    this.setState((state) => count: state.count + 1);
```

Reply:
But it is still valid!